client_context.py

```diff
@@ -1,13 +1,104 @@
+from __future__ import annotations
+
+from typing import Optional
+import json
+
 from edge_service import EdgeService
 from featurehub_repository import FeatureHubRepository
+from fh_state_base_holder import FeatureStateBaseHolder
+
+
+class StrategyAttributeCountryName:
```

Comment:
these are just strings in the end, so you may find a better way to do this in python, e.g. some kind of string enum or an array of strings where you check the values